pkg/formats/influx/influx.go

```diff
@@ -367,6 +369,66 @@ func (f *InfluxFormat) fromSnmpMetadata(in *kt.JCHF) []InfluxData {
 	return nil
 }
 
+func (f *InfluxFormat) fromKtranslate(in *kt.JCHF) []InfluxData {
+	// Map the basic strings into here.
+	attr := map[string]interface{}{}
+	metrics := map[string]kt.MetricInfo{"name": kt.MetricInfo{}, "value": kt.MetricInfo{}, "count": kt.MetricInfo{}, "one-minute": kt.MetricInfo{}, "95-percentile": kt.MetricInfo{}, "du": kt.MetricInfo{}}
+	f.mux.RLock()
+	util.SetAttr(attr, in, metrics, f.lastMetadata[in.DeviceName], false)
+	f.mux.RUnlock()
+	ms := make([]InfluxData, 0)
+	name := strings.ReplaceAll(in.CustomStr["name"], "chf.kkc.", "")
+
+	switch in.CustomStr["type"] {
+	case "counter":
+		if in.CustomStr["force"] == "true" || in.CustomBigInt["count"] > 0 {
+			ms = append(ms, InfluxData{
+				Name:        *Prefix + "ktranslate",
+				FieldsFloat: map[string]float64{name: float64(in.CustomBigInt["count"]) / 100},
+				Timestamp:   in.Timestamp * 1000000000,
+				Tags:        attr,
+			})
+		}
+	case "gauge":
+		if in.CustomStr["force"] == "true" || in.CustomBigInt["value"] > 0 {
+			ms = append(ms, InfluxData{
+				Name:        *Prefix + "ktranslate",
+				FieldsFloat: map[string]float64{name: float64(in.CustomBigInt["value"]) / 100},
+				Timestamp:   in.Timestamp * 1000000000,
+				Tags:        attr,
+			})
+		}
+	case "histogram":
+		if in.CustomStr["force"] == "true" || in.CustomBigInt["95-percentile"] > 0 {
+			ms = append(ms, InfluxData{
+				Name:        *Prefix + "ktranslate",
+				FieldsFloat: map[string]float64{name: float64(in.CustomBigInt["95-percentile"]) / 100},
+				Timestamp:   in.Timestamp * 1000000000,
+				Tags:        attr,
+			})
+		}
+	case "meter":
+		if in.CustomStr["force"] == "true" || in.CustomBigInt["one-minute"] > 0 {
+			ms = append(ms, InfluxData{
+				Name:        *Prefix + "ktranslate",
+				FieldsFloat: map[string]float64{name: float64(in.CustomBigInt["one-minute"]) / 100},
+				Timestamp:   in.Timestamp * 1000000000,
+				Tags:        attr,
+			})
+		}
+	case "timer":
+		if in.CustomStr["force"] == "true" || in.CustomBigInt["value"] > 0 {
+			ms = append(ms, InfluxData{
+				Name:        *Prefix + "ktranslate",
+				FieldsFloat: map[string]float64{name: float64(in.CustomBigInt["95-percentile"]) / 100},
```

Comment:
Should be `in.CustomBigInt["value"]`?

Reply:
Good catch -- it should be `95-percentile` in both places, not value:

https://github.com/kentik/ktranslate/blob/main/pkg/util/cmetrics/jchf.go#L75